### PR TITLE
Add null bubbling test in case of error

### PR DESCRIPTION
**Pull request by roman-kozachenko**

### CODE REVIEW

File: src/GraphQL.Tests/Errors/NullBubblingTests.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            public TestSubObjectWithNonNullableProperty()
+            {
+                Name = "Sub";
+                Field<StringGraphType>()
+                    .Name("one");
+
+                Field<NonNullGraphType<StringGraphType>>()
+                    .Name("two")
+                    .Resolve(_ => throw new Exception("wat"));
```
Comment (jquense) [inline src/GraphQL.Tests/Errors/NullBubblingTests.cs]: is this behavior for any error or the error in which it resolves to null but the value is non-nullable?

**Comment by joemcbride** [comment]
Tracking issue: #424

File: src/GraphQL.Tests/Errors/NullBubblingTests.cs (hunk 1 above)
Comment (roman-kozachenko) [reply src/GraphQL.Tests/Errors/NullBubblingTests.cs]: If 'two' fields is nullable, then it is ok to set it to null when error occurred.
But if it is non-nullable - nearest nullable parent should be set to null in result.

**Comment by roman-kozachenko** [comment]
I add simple bubbling.

**Comment by joemcbride** [comment]
@roman-kozachenko Appears to be working.  Once those tests are updated I can get this merged.

**Comment by roman-kozachenko** [comment]
@joemcbride I fixed tests.

File: src/GraphQL.Tests/Introspection/IntrospectionResult.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+        public static readonly string Data = "{\r\n  \"__schema\": null\r\n}";
```
Comment (joemcbride) [inline src/GraphQL.Tests/Introspection/IntrospectionResult.cs]: Hm, this change is a little troubling.  Didn't notice this one on the failed tests.  Will look at this closer.

Comment (roman-kozachenko) [reply src/GraphQL.Tests/Introspection/IntrospectionResult.cs]: You can find failed tests [here](https://ci.appveyor.com/project/graphql-dotnet-ci/graphql-dotnet/build/2.0.0.863-dxxhbeqj/tests). Failed test name is `GraphQL.Tests.Introspection.SchemaIntrospectionTests.validate_core_schema`.
As I see, `queryType` property is NonNullable, but its value is `null`. So, its value should be bubbled to nearest nullable parent - `__shema`.

**Comment by roman-kozachenko** [comment]
@joemcbride, @jquense any remarks for my PR? We want to have that changes on our production app, but until that PR will be merged we have to use custom nuget package based on my branch, losing ability to rapidly get your library updates.

**Comment by roman-kozachenko** [comment]
Since current pull request does not any feedback, I closed it.
All code is available in my branch https://github.com/roman-kozachenko/graphql-dotnet/tree/fix/null-bubbling. It can be merged in main repository, or any changes can be requested in future.

**Pull request closed without merging**